### DISCUSSION TITLE
Identifier Field in Record History

### DISCUSSION THREAD

**Discussion by ibte4691** [cortexrd/Knack-Toolkit-Library#368] [Q&A]
I assume the 'Display Field' from the updated record table is what is populating the 'Identifier' field in the Record History Table.  This doesn't appear to work if the Display Field is a text formula.  Is there a way around this?

**Comment by cortexrd** [comment]
You're right, this is something I've overseen and haven't yet implemented.  It makes the process more complicated because you can't have a Text Formula in a form.  

This means I'll to find the information elsewhere, typically in a Details view in the same page.

So, the keyword will also to be upgraded so we can specify the view and field of that Text Formula.

It's on the drawing board...

Norm

**Comment by ibte4691** [comment]
Thanks Norm.  Appreciate all the hard work.

**Comment by ibte4691** [comment]
By the way....I got around it by adding a short text field to the form
(display only) and used a record rule to populate that field with the text
formula field each time the form is submitted (or a record is added).  I
then changed my display field for the table to the new short text field.
